packages/webapp/src/containers/AddFarm/index.jsx

```diff
@@ -71,11 +71,19 @@ const AddFarm = () => {
     errors[COUNTRY]?.message ||
     errorMessage[errors[ADDRESS]?.type];
 
+  const showFarmNameCharacterLimitExceededError = () => {
+    setError(FARMNAME, {
+      type: 'manual',
+      message: t('ADD_FARM.FARM_NAME_ERROR'),
+    });
+  };
+
   const onSubmit = (data) => {
     const farmInfo = {
       ...data,
       gridPoints,
       farm_id: farm ? farm.farm_id : undefined,
+      showFarmNameCharacterLimitExceededError: showFarmNameCharacterLimitExceededError,
```

Comment:
I'm not sure about adding a function into a redux action payload. It seems to be working though.

Reply:
I took the reference of user sign-in when the user types a wrong password and from the API, we show the error message in the input field.